lib/systems/include/Controllers/SimpleController.h

```diff
@@ -28,8 +28,12 @@ class TorqueControllerSimple : public Controller
         writeout_.command = BaseControllerParams::TC_COMMAND_NO_TORQUE;
         writeout_.ready = true;
     }
-
+    /// @brief calculates torqe output based off max torque and simple torque scaling metric defined in constructor
```

Comment:
If it calculates the torque output, where does it put it? It clearly doesn't return anything-- does it simply update a member variable? If so, any changes to the object's state **must** be explicitly mentioned in the documentation.